src/content/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent.mdx

```diff
@@ -878,6 +751,101 @@ If everything is configured correctly and your data is being collected, you shou
+
+To enable the on-host integration log forwarding feature:
+
+<CollapserGroup>
+  <Collapser
+    id="elastic-search-logs"
+    title="Elasticsearch logs"
+  >
+    Copy (or rename) the `elasticsearch-log.yml.example` file to `elasticsearch-log.yml` to enable automatic Elasticsearch JSON formatted log parsing and forwarding to New Relic. No need to restart the agent.
+
+    **Example:**
+
+    ```
+    sudo cp /etc/newrelic-infra/logging.d/elasticsearch-log.yml.example /etc/newrelic-infra/logging.d/elasticsearch-log.yml
+    ```
+  </Collapser>
+
+  <Collapser
+    id="mysql-logs"
+    title="MySQL logs"
+  >
+    Copy (or rename) the `mysql-log.yml.example` file to `mysql-log.yml` to enable automatic MySQL error log parsing and forwarding to New Relic. No need to restart the agent.
+
+    **Example:**
+
+    ```
+    sudo cp /etc/newrelic-infra/logging.d/mysql-log.yml.example /etc/newrelic-infra/logging.d/mysql-log.yml
+    ```
+  </Collapser>
+
+  <Collapser
+    id="nginx-logs"
+    title="NGINX logs"
+  >
+    Copy (or rename) the `nginx-log.yml.example` file to `nginx-log.yml` to enable automatic NGINX access and error log parsing and forwarding to New Relic. No need to restart the agent.
+
+    **Example:**
+
+    ```
+    sudo cp /etc/newrelic-infra/logging.d/nginx-log.yml.example /etc/newrelic-infra/logging.d/nginx-log.yml
+    ```
+  </Collapser>
+
+  <Collapser
+    id="rabbitmq-logs"
+    title="Rabbitmq logs"
+  >
+    Copy (or rename) the `rabbitmq-log.yml.example` file to `rabbitmq-log.yml` to enable automatic Rabbitmq error log parsing and forwarding to New Relic. No need to restart the agent.
+
+    **Example:**
+
+    ```
+    sudo cp /etc/newrelic-infra/logging.d/rabbitmq-log.yml.example /etc/newrelic-infra/logging.d/rabbitmq-log.yml
+    ```
+  </Collapser>
+
+  <Collapser
+    id="redis-logs"
+    title="Redis logs"
+  >
+    Copy (or rename) the `redis-log.yml.example` file to `redis-log.yml` to enable automatic Redis error log parsing and forwarding to New Relic. No need to restart the agent.
+
+    **Example:**
+
+    ```
+    sudo cp /etc/newrelic-infra/logging.d/redis-log.yml.example /etc/newrelic-infra/logging.d/redis-log.yml
+    ```
+  </Collapser>
+</CollapserGroup>
+
+## Enable log forwarding on agent installed using Linux tarball [#tarball-install]
+
+Our custom Linux installation process for infrastructure monitoring allows you to tailor all aspects of the installation process, and to place files and folders on your machine. If you choose the [assisted](https://docs.newrelic.com/docs/infrastructure/install-infrastructure-agent/linux-installation/tarball-assisted-install-infrastructure-agent-linux) or [manual](https://docs.newrelic.com/docs/infrastructure/install-infrastructure-agent/linux-installation/tarball-manual-install-infrastructure-agent-linux) tarball installation process, follow these steps to implement the log forwarder feature:
```

Comment:
I think the links are formatted incorrectly but I'm not sure if they need to be full or start at /docs/ for some specific reason.